docs/2-resource-customization.md

```diff
@@ -227,6 +228,14 @@ end
 Note that the "Blog" parent menu item doesn't even have to exist yet; it can be
 dynamically generated for you.
 
+To further nest an item under a submenu, provide an array of parents.
+
+```ruby
+ActiveAdmin.register Post do
+  menu parent: ["Admin", "Blog"]
```

Comment:
@taralbass  does it mean array of only 2 elements supported? or it supports endless-nesting

Reply:
@Fivell It will support arbitrarily deep nesting, including styling, EXCEPT that if nested deeply enough, a menu will eventually "fall off" the right side of the window, and I did not make any attempt to handle that.